dockerfiles/nodejs-21.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:21.7-alpine3.19
+
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="package.json,package-lock.json"
+
+WORKDIR /app
+
+COPY package.json ./
+COPY package-lock.json ./
+
+RUN npm i 
```

Comment:
Let's use the version that doesn't update package-lock.json, I think it's called `npm ci`